### ISSUE TITLE
ng build ssr error

### ISSUE THREAD

**Issue by nbiles** [nomadreservations/ngx-codemirror#1]
So far I love the module but I am having an issue with ssr prod build.

```
ERROR in ./node_modules/@nomadreservations/ngx-codemirror/ngx-codemirror.ngfactory.js
Module not found: Error: Can't resolve 'ngx-codemirror' in '/path/to/project/node_modules/@nomadreservations/ngx-codemirror'
resolve 'ngx-codemirror' in '/path/to/project/node_modules/@nomadreservations/ngx-codemirror'
  Parsed request is a module
  using description file: /path/to/project/node_modules/@nomadreservations/ngx-codemirror/package.json (relative path: .)
    Field 'browser' doesn't contain a valid alias configuration
  after using description file: /path/to/project/node_modules/@nomadreservations/ngx-codemirror/package.json (relative path: .)
    resolve as module
      /path/to/project/node_modules/@nomadreservations/ngx-codemirror/node_modules doesn't exist or is not a directory
      /path/to/project/node_modules/@nomadreservations/node_modules doesn't exist or is not a directory
      /path/to/project/node_modules/node_modules doesn't exist or is not a directory
      /node_modules doesn't exist or is not a directory
      looking for modules in /path/to/project/node_modules
        using description file: /path/to/project/package.json (relative path: ./node_modules)
          Field 'browser' doesn't contain a valid alias configuration
        after using description file: /path/to/project/package.json (relative path: ./node_modules)
          using description file: /path/to/project/package.json (relative path: ./node_modules/ngx-codemirror)
            no extension
              Field 'browser' doesn't contain a valid alias configuration
              /path/to/project/node_modules/ngx-codemirror doesn't exist
            .ts
              Field 'browser' doesn't contain a valid alias configuration
              /path/to/project/node_modules/ngx-codemirror.ts doesn't exist
            .js
              Field 'browser' doesn't contain a valid alias configuration
              /path/to/project/node_modules/ngx-codemirror.js doesn't exist
            as directory
              /path/to/project/node_modules/ngx-codemirror doesn't exist
```

It appears that this is only happening when I import into a child, shared, or lazy loaded module.

**Comment by nbiles** [comment]
So...I temporarily solved this by creating a file in my src root: 

/path/to/project/src/ngx-codemirror.js
```
export * from '@nomadreservations/ngx-codemirror';
```

**Comment by paullryan** [comment]
Ya it's been throwing me in a few of my project right now I'm looking into it from a number of angles as I'm trying to release some things for our first client. I'll see if I can fix this while I'm at it.

**Comment by paullryan** [comment]
Fixed in 1.0.0-rc.3

**Comment by paullryan** [comment]
@nbiles Please note that codemirror is now a peer of this project.

**Comment by gnujeremie** [comment]
Can you push the last version to npm ?
Thanks !

**Comment by paullryan** [comment]
@gnujeremie it is on npm but it's tagged as @next so please `npm install @nomadreservations/ngx-codemirror@1.0.0-rc.3`

**Comment by paullryan** [comment]
Just published 1.0.0 as @lastest so you can use that now @gnujeremie

**Comment by gnujeremie** [comment]
awesome, thanks :)
